changes.go

```diff
@@ -13,10 +13,13 @@ import (
 
 // Changes returns the changes stream for the database.
 func (d *db) Changes(ctx context.Context, opts map[string]interface{}) (driver.Changes, error) {
-	overrideOpts := map[string]interface{}{
-		"feed":      "continuous",
-		"since":     "now",
-		"heartbeat": 6000,
+	var overrideOpts map[string]interface{}
+	if opts["feed"] != "normal" {
+		overrideOpts = map[string]interface{}{
+			"feed":      "continuous",
+			"since":     "now",
+			"heartbeat": 6000,
+		}
```

Comment:
I think it would be better to remove this 'overrideOpts' thing. It's better to have the user provide all desired options.